editor.planx.uk/src/routes/subscription.tsx

```diff
@@ -0,0 +1,26 @@
+import { compose, mount, route, withData } from "navi";
+import { Subscription } from "pages/FlowEditor/components/Subscription/Subscription";
+import React from "react";
+
+import { makeTitle } from "./utils";
+
+const subscriptionRoutes = compose(
+  withData((req) => ({
+    mountpath: req.mountpath,
+  })),
+
+  mount({
+    "/": compose(
```

Comment:
We need to implement an auth guard here to stop non-teamAdmin users navigating directly to this page.

A non-teamAdmin user can access the following - 

<img width="491" height="292" alt="Image" src="https://github.com/user-attachments/assets/421925dc-ba73-4a31-8717-a1233e1b373a" />